ZeldaFullEditor/Gui/DungeonMain.cs

```diff
@@ -1623,6 +1623,40 @@ public void entrancetreeView_AfterSelect(object sender, TreeViewEventArgs e)
 				}
 			}
 
+			this.setEntranceProperties(en, en.CameraTriggerX, en.CameraTriggerY);
+		}
+
+		/// <summary>
+		///		Called when moving the mouse in the "set entrance position" mode.
+		/// </summary>
+		/// <param name="mouseX"> The adjusted X mouse position. </param>
+		/// <param name="mouseY"> The adjusted Y mouse position. </param>
+		public void entrancetreeView_AfterSelect(int mouseX, int mouseY)
```

Comment:
Magic numbers in this routine should be documented at least if not made a constant.

Reply:
good idea, I think I'll only leave a comment documenting them because they are only used here and a very specific to this function.